internal/acctest/authentication/oauth_test.go

```diff
@@ -0,0 +1,64 @@
+package auth_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/provider"
+)
+
+func getOAuthEnvVars() {
```

Comment:
This method doesn't appear to do anything, it just retrieves the vars but doesn't save them anywhere